2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -19,7 +19,33 @@
 //cspell: enable
 
 function createBookList(books) {
-  // your code goes in here, return the ul element
+  const ul = document.createElement('ul');
+  for (const item of books) {
+    const newPar = document.createElement('p');
+    newPar.innerText = `${item.title} - ${item.author}`;
+    const li = document.createElement('li');
+    li.appendChild(newPar);
+
+    const img = document.createElement('img');
+    if (item.title === 'The Design of Everyday Things') {
+      img.src = 'assets/the_design_of_everyday_things.jpg';
+    } else if (item.title === 'The Most Human Human') {
+      img.src = 'assets/the_most_human_human.jpg';
+    } else if (item.title === 'The Pragmatic Programmer') {
+      img.src = 'assets/the_pragmatic_programmer.jpg';
+    }
+    img.alt = item.title;
+    li.appendChild(img);
+
+    if (item.alreadyRead === false) {
+      li.style.backgroundColor = 'red';
+    } else {
+      li.style.backgroundColor = 'green';
+    }
+
+    ul.appendChild(li);
+  }
+  return ul;
 }
 
 const myBooks = [
```

Comment:
It is maybe more intuitive to have this const declared above the method since you use it in the method